app/controllers/reviews_controller.rb

```diff
@@ -1,2 +1,16 @@
 class ReviewsController < ApplicationController
+  def new
+    @reviews = Review.all
+  end
+
+  def create
+    @review = Review.find(params[:review_id])
+
+    if @review.save
+      flash[:success] = "Review was created successfully"
+      redirect_to product_path(@reviews)
```

Comment:
redirect_to product_path(@reviews)
Might need to send in product _id instead you should be able to know this from the review